index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/7d6392431b.js" crossorigin="anonymous"></script>
+    <title>Apple Clone</title>
+</head>
+
+<body>
+    <main>
+        <nav class="navbar">
+            <ul class="nav-list">
+                <li class="nav-list-item">
+                    <a class="anchor-list-item apple-logo" href="#"><i class="fab fa-apple"></i></a>
+                </li>
+                <li class="nav-list-item">
+                    <a class="anchor-list-item store" href="#">Store</a>
+                </li>
+                <li class="nav-list-item">
+                    <a class="anchor-list-item mac" href="#">Mac</a>
+                </li>
+                <li class="nav-list-item">
+                    <a class="anchor-list-item ipod" href="#">iPod</a>
+                </li>
+                <li class="nav-list-item">
+                    <a class="anchor-list-item iphone" href="#">iPhone</a>
+                </li>
+                <li class="nav-list-item">
+                    <a class="anchor-list-item ipad" href="#">iPad</a>
+                </li>
+                <li class="nav-list-item">
+                    <a class="anchor-list-item itunes" href="#">iTunes</a>
+                </li>
+                <li class="nav-list-item">
+                    <a class="anchor-list-item support" href="#">Support</a>
+                </li>
+                <li class="nav-list-item">
+                    <div class="search-input">
+                        <i class="fas fa-search"></i>
+                        <input class="box-input" type=text>
+                    </div>
+                </li>
+            </ul>
+        </nav>
+        <header>
+            <div class="headings">
+                <h1>
+                    <img src="img/your_verse_title_small.png" alt="What will your verse be.">
+                </h1>
+                <h2>
+                    <img src="img/your_verse_subhead_small.png" alt="Everyone has something to share">
+                </h2>
+                <h3>
+                    <img src="img/cta_your_verse_27pt.png" alt="something here">
+                </h3>
+            </div>
+        </header>
+    </main>
+    <aside>
+        <img class="grid-item iphone5s" src="img/promo_iphone5s.jpg" alt="iphone 5S">
```

Comment:
- [ ] Each image in the aside should be wrapped inside an `<a>` tag.
![Screenshot_2020-03-23 dist html_css building_with_backgrounds_and_gradients md · master · Microverse guides tse code_review](https://user-images.githubusercontent.com/16193212/77315676-b5682280-6ce6-11ea-8929-b3d43bc7cc1d.png)